sensor/common/registry/registry_store.go

```diff
@@ -46,7 +60,7 @@ type Store struct {
 
 	mutex sync.RWMutex
 
-	checkTLS CheckTLS
+	checkTLSFn CheckTLS
```

Comment:
The go convention is `Func` instead of `Fn`

Reply:
https://github.com/stackrox/stackrox/pull/7962/commits/642be4031dd69b7d9eb121b7b0243c43c31b6e71

How did you conclude this is the go convention? The go standard library uses both conventions ([Fn example](https://github.com/golang/go/blob/e47cab13a5acfb99ed6e62eeb51772af94c5f526/src/net/dnsclient_unix.go#L656-L657), [Func example](https://github.com/golang/go/blob/3a69dcdc9f487f79fdce82536c97b49ba7f216c7/src/net/hook_unix.go#L17-L20)) as well as stackrox ([Fn example](https://github.com/stackrox/stackrox/blob/0bbf61d6137fa8a269d25523402160353d6bf03d/central/administration/events/handler/handler_test.go#L50), [Func example](https://github.com/stackrox/stackrox/blob/c538c739cc4e3bcf32ec826b675a46d8e3f91693/central/compliance/aggregation/aggregation.go#L313))

Reply:
great question: I was told to use `func` over `fn`, so I've been sticking with that. Maybe this is something that belongs in our styleguide, if it doesn't exist yet